apis/addons/v1alpha1/addonoperators_types.go

```diff
@@ -26,6 +26,21 @@ type AddonOperatorSpec struct {
 	// when set to True
 	// +optional
 	Paused bool `json:"pause"`
+
+	// OCM specific configuration.
+	// +optional
```

Comment:
Can you pls add a comment that states that this key will not be optional after the migration?